system/ee/legacy/libraries/simplepie/SimplePie/File.php

```diff
@@ -109,7 +109,7 @@ public function __construct($url, $timeout = 10, $redirects = 5, $headers = null
 				curl_setopt($fp, CURLOPT_REFERER, $url);
 				curl_setopt($fp, CURLOPT_USERAGENT, $useragent);
 				curl_setopt($fp, CURLOPT_HTTPHEADER, $headers2);
-				if (!ini_get('open_basedir') && !ini_get('safe_mode') && version_compare(SimplePie_Misc::get_curl_version(), '7.15.2', '>='))
```

Comment:
The posts on internet are saying `CURLOPT_FOLLOWLOCATION` cannot be used when either `open_basedir` or `safe_mode` are set, but this might have changed? Making the change to keep it consistent with EE5